contrib/sssd.spec.in

```diff
@@ -701,6 +703,8 @@ do
     cat $subpackage.lang
 done
 
+install -D -p -m 0644 %{SOURCE1} %{buildroot}%{_sysusersdir}/sssd.conf
```

Comment:
Is this name - 'sssd.conf' - mandated anywhere?
It's confusing because it matches name of SSSD config file.
Would it be possible to use different name for a file with sysuser definition?

Reply:
I checked `_sysusersdir` location (/usr/lib/sysusers.d/) and I only saw one exception, which I guess makes it valid to use another extension

Reply:
Per `man sysusers.d`
~~~
CONFIGURATION DIRECTORIES AND PRECEDENCE
       Each configuration file shall be named in the style of package.conf or package-part.conf. The second variant should be used when it is desirable to make it easy to override just this part of configuration.
~~~